atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -506,14 +506,21 @@ def get_build(self, metadata):
         if not isinstance(source, GitSource):
             raise RuntimeError('git source required')
 
-        extra = {'image': {'autorebuild': is_rebuild(self.workflow)}}
+        extra = {
+            'image': {'autorebuild': is_rebuild(self.workflow)},
+            'submitter': session.getLoggedInUser().get('name'),
+        }
 
+        koji_task_owner = None
         koji_task_id = metadata.get('labels', {}).get('koji-task-id')
         if koji_task_id is not None:
             self.log.info("build configuration created by Koji Task ID %s",
                           koji_task_id)
             try:
-                extra['container_koji_task_id'] = int(koji_task_id)
+                extra['container_koji_task_id'] = koji_task_id = int(koji_task_id)
+                koji_task_owner = get_koji_task_owner(
+                    session, koji_task_id, default=dict(name=None)
```

Comment:
Oh interesting how `default` is very misleading in this function. Can we change `get_koji_task_owner` to return an empty dict if owner info can't be found?

```python
def get_koji_task_owner(session, task_id, default=None):
  default = {} if default is None else default
  ...
```